### PR TITLE
Added low-level build to Travis

### PR DESCRIPTION
**Pull request by VivaanBahl**
Added the compilation of low-level to the Travis CI build

Essentially just calls the `Makefile`, no static analysis has been defined yet

### CODE REVIEW

File: real_time/arduino_src/radio_buggy_mega/fingerprint.py [hunk 1]
```diff
@@ -31,7 +31,7 @@ def main():
     #Gets current branch name
     branch = "unnamed branch"
     try:
-        branch = str(subprocess.check_output(["git", "symbolic-ref", "--short", "HEAD"])).replace("\n", "")
+        branch = str(subprocess.check_output(["git", "log", "--pretty=format:'%h'", "-n", "1"])).replace("\n", "")
```
Comment (ramensandwich) [inline real_time/arduino_src/radio_buggy_mega/fingerprint.py]: This retrieves the hash, not the branch name

Comment (grypoB) [inline real_time/arduino_src/radio_buggy_mega/fingerprint.py]: Maybe try: `git rev-parse --abbrev-ref HEAD`

File: real_time/arduino_src/radio_buggy_mega/fingerprint.py [hunk 2]
```diff
@@ -1,7 +1,7 @@
-""" fingerprint.py:
```
Comment (ramensandwich) [inline real_time/arduino_src/radio_buggy_mega/fingerprint.py]: Why are so many unrelated lines on this file touched?

Comment (grypoB) [inline real_time/arduino_src/radio_buggy_mega/fingerprint.py]: seems to be only end of line spaces which were removed. Should be fine

Comment (ramensandwich) [inline real_time/arduino_src/radio_buggy_mega/fingerprint.py]: It adds unnecessary diffs and makes the "blame" command useless on Github though

Comment (grypoB) [inline real_time/arduino_src/radio_buggy_mega/fingerprint.py]: This raise the issue of trailing white spaces all over the repo. What do we do with them?
We have the following options:
* changing them, adding a diff, making the blame command more complicated to use
  * by making a "clean-up" pull request with tons of this white spaces removal
  * or doing like Vivaan is doing now. Removing them as he sees them, even though it is unrelated to the diff.
* keeping them in the long term. Removing them only if we work on the related lines. Less diffs, less pretty code

About the diffs number, if he reverts his changes on those line and we keep track of all the history on master, it is still going to keep the diffs, making the blame command less useful. We might think about merging branches by collating diffs into a single one (for small changes like this branch).

Comment (ramensandwich) [inline real_time/arduino_src/radio_buggy_mega/fingerprint.py]: I don't think that trailing whitespaces are necessarily ugly. My vote is to leave them as they are to avoid unnecessary diffs for now, but when future changes are getting merged in, we should avoid including them

**Comment by ramensandwich** [comment]
Looks much cleaner.  How well has the travis build been tested (i.e. successfully building on good code and failing when appropriate)?  Once you're happy with the Travis script and have tested the fingerprint script on the buggy and verify that everything still works, I'll approve this.

**Comment by VivaanBahl** [comment]
I haven't tested it much, been working on other stuff. I'll probably get it done once we get back
